check-for-travis.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+require 'octokit'
+
+client = Octokit::Client.new :access_token => ENV['GITHUBTEACHER_TOKEN'] || raise("You need a GitHub Teacher access token")
+
+# Get the tree for a sha
+tree = client.tree("githubschool/on-demand-github-pages", "f1b10e926e7ace9862bf65b789204f22cccba1e0")
+
+# Select .travis.yml from the tree
+travis_yml = tree.tree.select { |t| t.path == ".travis.yml" }
+
+# Get the corresponding blob for the .travis.yml sha
+blob = client.blob("githubschool/on-demand-github-pages", travis_yml.first.sha)
+
+# Inspect the content
+puts Base64.decode64(blob.content)
```

Comment:
Been working on that.

```rb
#!/usr/bin/env ruby
require 'octokit'
require 'digest'

client = Octokit::Client.new :access_token => ENV['GITHUBTEACHER_TOKEN'] || raise("You need a GitHub Teacher access token")

# Get the tree for a sha
tree = client.tree("githubschool/on-demand-github-pages", "7e88ef3d11f84ed351bf1e0d733fef7cbe54fc18")

# Select .travis.yml from the tree
travis_yml = tree.tree.select { |t| t.path == ".travis.yml" }

puts travis_yml.inspect

# Get the corresponding blob for the .travis.yml sha
blob = client.blob("githubschool/on-demand-github-pages", travis_yml.first.sha)

travis_yml_sha256 = Digest::SHA256.new
travis_yml_sha256.update Base64.decode64(blob.content)

# TODO: find commit's .travis.yml content and `sha` to compare to `travis_yml_sha256`

current_cimmit_sha = "bb8c9e186f687ca3af294dbb2bf870040e837766" # TODO: we need the current commit's `sha`
current_pr_id = 2                                               ## TODO: we need the current commit's `id`

if current_cimmit_sha != travis_yml_sha256.hexdigest
  # client.create_status "githubteacher/practice-sinatra", current_cimmit_sha, "failure", :description => "You are not allowed to change `.travis.yml`, please reveret your changes to `.travis.yml` and commit again."
  #
  # client.add_comment "githubteacher/practice-sinatra", current_pr_id, "You are not allowed to change `.travis.yml`, please reveret your changes to `.travis.yml` and commit again."

  puts ".travis.yml changed !!"
end
```

Something like that maybe.
We need 
- the commit `sha`
- the committed `.travis.yml` content to SHA256 for comparison
- the original `.travis.yml` content we want to set as ungangeable
- the original `.travis.yml` SHA256 for comparison